nextcloud_update.sh

```diff
@@ -308,15 +331,18 @@ https://shop.hanssonit.se/product/upgrade-between-major-owncloud-nextcloud-versi
     exit 1
 fi
 
-# Check if new version is larger than current version installed.
-print_text_in_color "$ICyan" "Checking for new Nextcloud version..."
-if version_gt "$NCVERSION" "$CURRENTVERSION"
+# Check if new version is larger than current version installed. Skip version check if you want to upgrade to a prerelease.
+if [ -z "$PRERELEASE_VERSION" ]
 then
-    print_text_in_color "$ICyan" "Latest release is: $NCVERSION. Current version is: $CURRENTVERSION."
-    print_text_in_color "$IGreen" "New version available, upgrade continues!"
-else
-    print_text_in_color "$IGreen" "You already run the latest version! ($NCVERSION)"
-    exit 0
+    print_text_in_color "$ICyan" "Checking for new Nextcloud version..."
+    if version_gt "$NCVERSION" "$CURRENTVERSION"
```

Comment:
I don't think that it would update you to a version below your actual one because of this check.
So as I understand it, it would only update you automatically if a release is added to the stable directory.

Reply:
So eg if only 18.0.4 is found on the stable directory, ncversion would always be less then 19.0.0RC1, aka currentversion and hence not upgrading.
it would only upgrade automatically if e.g. 19.0.1 is released to the stable branch.

Reply:
Problem here is that `19.0.0RC1` isn't a pure number as it contains `RC1` so I don't know what will happen to be honest.

Sure, 19 is higher than 18, but if the check fails, maybe it skips it completely anyway?

Reply:
the `version` function is made to work with numbers, nothing else. Hence I'm worried it might fail.

Reply:
actually I've tried that for you and although the test instance is now on 19.0.0RC1, currentversion:
`CURRENTVERSION=$(sudo -u www-data php $NCPATH/occ status | grep "versionstring" | awk '{print $3}')`
returns only 19.0.0 on my test instance, which should not make any problem with version_gt.

Reply:
OK then, let's merge this.